### ISSUE TITLE
Brain storming session before Friday real-life meeting.

### ISSUE THREAD

**Issue by swom** [tresinformal/game#12]
Hello guys,
I think that, before pitching our own games and decide what we are going to do, it would be nice to have some more brainstorming. I would like to meet next week before Friday. We could meet at Zernike or at someone's place ,have a few beers, play a few games we like, and see if we can come up with some cool concepts. Afterwards, we can all recollect our thoughts and come with our game pitch for Friday. How does that sound? 
As Boris is already doing (#3), I invite everybody to share videos or links to games we like and could be inspiring (let's do it on #3 ).

**Comment by boman0815** [comment]
ill not be here next friday. anyways i have some videogames planned for tonight, we could meet at my place in interested .. cheers, bo

**Comment by boman0815** [comment]
sorry for my confusing comment, ifyou wanna join for some games today (friday) ill most likely be playing at home with some friends. 

anyway as stefano suggested we could have a more serious (not richel serious ;-)) brainstorming sessoin an afternoon early next week. thuesday? i can bring my nintendo switch for some games after.

**Comment by richelbilderbeek** [comment]
Great initiative!

I enjoy the meetings I chair are dubbed 'Richel serious'! Meetings are there to make decisions and should be short :muscle:. In case of doubt, I think the chairman should be elected democratically if there are different ideas of how a meeting should be chaired.

**Comment by swom** [comment]
Hi Richel,

I agree with you that meetings should be short and to make decisions. I would think of this more as a pre-meeting, where we can  gather more informations to then make a more informed choiced, and be more all on the same track. Not everybody needs to attend, and no decisions will be made.
For the rest, I think, you should keep charing the meetings, as per usual, since you have the most experience and expediency in this kind of things.

**Comment by richelbilderbeek** [comment]
Cool, seems we are on the same page here :+1:. See you soon, probably already tomorrow :see_no_evil:

**Comment by boman0815** [comment]
richel, for me its also good if you stay chairman. richel serious is a good thing! 
are we meeting tuesday afternoon? informally i mean

**Comment by richelbilderbeek** [comment]
> are we meeting tuesday afternoon? informally i mean

I suggest you, @swom and those interested plan the brainstorming meeting at your leisure. 

Personally, I will be absent, as I have no interest in deciding what game we'll make. And I can permit to do: apparently there are some people in the team who do invest the time to come up with an awesome plan :grin:

**Comment by swom** [comment]
So,
I propose Tuesday(tomorrow), at 7 pm(some members will be busy before that time). I will book Franjo's Fancy library or we could do it in our coffee room (bigger TV screen).
I will also create a facebook group to remind peeps to check GitHub.

**Comment by boman0815** [comment]
ill leave zernike soon. if anyone is in for gaming after brainstorming tomorrow i will bring my nintendo stuff. please send me a message if i should do it (facebook, whatsapp -->(+491778375671))

cheers,
 bo

**Comment by marinapapa** [comment]
Switch to 5pm?

**Comment by swom** [comment]
Hello peeps,
I made a sort not-so-short resume of the ideas we came up with today. Is on a google doc, (sorry Richel, I am sure there is a better way to this on GitHub).
If you want add stuff to the ideas block use suggestion mode so we can see who propose the idea. If you wanna comment, feel free to comment!
We could use this doc to review the ideas at hand before deciding on Friday, we can either decide one of these ideas or use these ideas as inspiration for even cooler ideas to be proposed on Friday!
Here is the [link to the google doc](https://docs.google.com/document/d/1hNQfrlprKvZVrt4qWT3pBTB_8rSDlh_caX-eEwK-SMU/edit?usp=sharing)

**Comment by richelbilderbeek** [comment]
Cool! 

I've collected the seven ideas:

 * Bird nest defense and reproduction game, with evolution in it!
 * Phd Game
 * The useless game
 * Limited resources mechanic
 * 2D PvP chasing game
 * Defense game, Guards vs. Thieves
 * Turn based brawler/extreme-volleyball game.

Just remember: on Friday everyone will have 3 minutes max **per person** (yes, not per suggestion!) to talk about the idea.

Personally, I enjoyed most ideas, and I am pretty we'll vote for the coolest games :+1:
